### PR TITLE
[tsp-client] Fix bundled dependency issue

### PR DESCRIPTION
**Pull request by tjprescott**
Includes openapi-diff as bundled dependency in package.json.

### CODE REVIEW

File: tools/tsp-client/package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "tsp-client": "cmd/tsp-client.js"
   },
   "files": [
+    "openapi-diff-1.0.0.tgz",
```
Comment (catalinaperalta) [inline tools/tsp-client/package.json:36]: Do we still need this?

Comment (tjprescott) [reply tools/tsp-client/package.json:36]: Nope

File: tools/tsp-client/package.json [hunk 2]
```diff
@@ -71,6 +72,9 @@
     "extension": [
       "ts"
     ],
+    "bundleDependency": [
```
Comment (catalinaperalta) [inline tools/tsp-client/package.json:75]: Assuming this one does need to stay, right?

**Review by catalinaperalta — APPROVED** [review]
Can you also add a changelog entry for the new release?

**Pull request merged**